crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -456,7 +458,25 @@ scalar MoveTypeSignature
 type MoveValue {
 	type: MoveType!
 	bcs: Base64!
+	"""
+	Structured contents of a Move value.
+	"""
 	data: MoveData!
+	"""
+	Representation of a Move value in JSON, where:
+	
+	- Addresses and UIDs are represented in canonical form, as JSON strings.
+	- Bools are represented by JSON boolean literals.
+	- u8, u16, and u32 are represented as JSON numbers.
+	- u64, u128, and u256 are represented as JSON strings.
```

Comment:
This is different from how the structured representation handles numbers, (where they all get turned into `BigInt`s).  Originally, I set it up this way because I thought it would be annoying to work with to have two (or more) variants for numbers in the structured case, and that's not an issue here (also this is the pattern that's used in the various BCS to JSON libraries, so I thought it was worth replicating here), but let me know your thoughts.

Reply:
I think this should be consistent, out BigInt scalar type should also result in a JSON string in the response

Reply:
To make sure I understand, are you saying that this type should also change so that `u8` to `u32` also produce JSON strings to be consistent with `BigInt`?

Reply:
no, just saying that the BigInt scalar used in other places would also produce JSON strings since graphql responses are just JSON, so I think the current behavior is consistent.  Nothing to change here

Reply:
I think I misunderstood your original comment.  I  hadn't realized all number types were BigInts, sorry for the confusion.  This still looks good to me as is.

Reply:
The inconsistency here is one level up from that: if you ask for the same content as `data` rather than `JSON` then a `u8` will show up as a `{ "Number": BigInt }` in the `data` representation and as just a `number` in the `JSON` representation, but if you're happy, I'm happy!